src/main/resources/jenkins/branch/ItemColumn/column.jelly

```diff
@@ -25,26 +25,36 @@ THE SOFTWARE.
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <td style="${indenter.getCss(job)}">
+  <d:taglib uri="local">
+    <d:tag name="link">
+      <a href="${jobBaseUrl}${job.shortUrl}" class='model-link jenkins-table__link'>
+        <j:choose>
+          <j:when test="${it.getTitle(job) == null or h.getRelativeDisplayNameFrom(job, itemGroup).equals(it.getTitle(job))}">
+            <l:breakable value="${h.getRelativeDisplayNameFrom(job, itemGroup)}"/>
+          </j:when>
+          <j:otherwise>
+            ${it.getTitle(job)}
+            <span style="opacity: 0.75; margin-left: 0.625rem; white-space: nowrap;"><l:breakable value="${h.getRelativeDisplayNameFrom(job, itemGroup)}"/></span>
```

Comment:
would be good if we could drop the `PR-` and convert it to a `#` Would be simpler and consistent with how GitHub displays it